mipengine/controller/controller.py

```diff
@@ -66,7 +66,7 @@ def run_algorithm_executor_in_threadpool(
         loop = asyncio.get_running_loop()
 
         # DEBUG(future logging..)
-        ctrl_logger.getLogger(__name__).debug(
+        ctrl_logger.getAppLogger().debug(
             f"\n(controller.py::exec_algorithm) starts executing-> "
             f"{algorithm_name=} with {context_id=}\n"
         )
```

Comment:
```
logger.debug(f"starts executing->  {algorithm_name=} with {context_id=}")
```
the (conntroller.py::exec_algorithm) etc do not need to be in the message anymore as they are anyway put in there by the logger now